extensions/wikia/UserLogin/templates/UserLoginSpecial_Providers.php

```diff
@@ -1,4 +1,4 @@
-	<div class="sso-login">
+	<div class="sso-login fb-loaded hidden">
```

Comment:
You have a good point about it being misleading. I did this to separate styling class (i.e. `.sso-login`) from the functional class `fb-loaded`. Rename?